cmd/boostd/dagstore.go

```diff
@@ -309,3 +311,46 @@ var dagstoreDestroyShardCmd = &cli.Command{
 		return nil
 	},
 }
+
+var dagstoreLookupCmd = &cli.Command{
+	Name:      "lookup-cid",
```

Comment:
wouldn't you want to be more explicit that you are looking up a piece cid?  `lookup-piece-cid`?

Reply:
The description provides the functionality details. I would prefer to keep command name to be short.

Reply:
I agree with @LaurenSpiegel that it'd be better to be explicit here as `cid` it very generic. There is the `alias` parameter that you could use to set a short command alias (such as `lpc` or something else).